docs/configuration/downstream/koji_build.md

```diff
@@ -77,8 +87,16 @@ Packit dist-git configuration.
      (don't forget to put the group name in quotes - a token in YAML is not allowed to start with `@`)
    - `all_admins` alias - allowing all users with admin access to the dist-git repo
    - `all_committers` alias - allowing all users with commit access to the dist-git repo
+* **sidetag_group** - name of a sidetag group this Koji build should be tagged into.
+  The name has to be unique across all projects that use Packit, so don't hesitate to be descriptive.
+  If specified, after a build finishes it is tagged into a sidetag (that is created if it doesn't already exist)
+  corresponding to the dist-git branch of the build.
+* **dependencies** - list of downstream package names that are required to be tagged
+  into the configured sidetag group for this job to be triggered.
+* **dependents** - list of downstream package names that should react to tagging of this build
+  into the configured sidetag group.
```

Comment:
I think this are nicely written, but to me they are really hard to grasp. Can we add an example?
```suggestion
* **dependencies** - list of downstream package names that are required to be tagged
  into the configured sidetag group for this job to be triggered. As an example the `packit` package depends  on the `python-specfile` package. Hence in `packit.yaml` inside `packit` project you find `python-specfile` package listed in the `dependencies`.
* **dependents** - list of downstream package names that should react to tagging of this build
  into the configured sidetag group. As an example the `packit` package depends  on the `python-specfile` package. Hence in `packit.yaml` inside `specfile` project you find `packit` package listed in the `dependents`.
```

Reply:
What about links to the guide/examples as Franta suggests?

Reply:
The links are useful, and I would take them, but personally if I small example can help I prefer having it instead of having to read a long guide.

Reply:
I've added examples, what do you think?